gateway/operations/deleteobjects.go

```diff
@@ -15,8 +15,13 @@ import (
 
 type DeleteObjects struct{}
 
-func (controller *DeleteObjects) RequiredPermissions(_ *http.Request, _ string) ([]permissions.Permission, error) {
-	return nil, nil
```

Comment:
So we never supported delete objects? Or did we allow it for all users?

Reply:
Never supported.